src/TeleportJoin.sol

```diff
@@ -19,7 +19,12 @@ pragma solidity 0.8.14;
 import "./TeleportGUID.sol";
 
 interface VatLike {
+    function wards(address) external view returns (uint256);
+    function rely(address) external;
+    function init(bytes32) external;
+    function file(bytes32 ilk, bytes32 what, uint data) external;
```

Comment:
I'd suggest to create a new `VatLike` in the test to avoid adding not used methods in the contract file. In general we try to keep that practice.
You'll need to change the import in the test to just bring the `TeleportJoin` to be able to define `VatLike` though.